### ISSUE TITLE
Won't compile on Raspbian because of 'may 2005'

### ISSUE THREAD

**Issue by sheosi** [snipsco/rustling-ontology#209]
Hello, rustling-ontology fails to compile on Raspbian because the example of 'may 2005' fails on German, and later o English, Spanish and French.

This is the error returned:

`thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ErrorMessage { msg: "example: \"mai 2005\" matched no rule" }', /root/.cargo/git/checkouts/rustling-ontology-a5f364cfd4d376e4/3bb1313/build.rs:45:17`

This happens both when installing snips-nlu by pip, and when compolling a program with snips-nlu-rs. I've 'workarounded' the later by overriding the dependency and removing that example, but I don't how to do that on pip.

**Comment by Flocksserver** [comment]
Same problem here. Any ideas why that happened?

**Comment by sheosi** [comment]
Not really. AFAIK that only seems to trigger on 32-bits OSs since on my new
Ubuntu install I've had 0 problems so far.

**Comment by Flocksserver** [comment]
there is a PR commenting out the test lines but that thats a quick workaround, nothing more than that I Think... but yeah on my mac and ubuntu x68 intel architectures -> no problems. In the next days I will try compiling rustling ontology on my other pi

 description: Computer
    product: Raspberry Pi 4 Model B Rev 1.1
    width: 64 bits

64bits but arm

**Comment by sheosi** [comment]
Then I dont know whats changed, other than using Ubuntu. Commenting out the
lines works as far as i remember, but i ran into other problems. Also, with
raspbian 64 bits, I think I've read somewhere that while the kernel is 64
bits the userland is 32, and thus the compilation is defaulting to 32,
can't say for sure though.
